src/Microsoft.Diagnostics.Monitoring.StartupHook/ParameterCapturing/Pipeline/ParameterCapturingPipeline.cs

```diff
@@ -206,6 +206,17 @@ public void SubmitRequest(StartCapturingParametersPayload payload, IFunctionProb
             }
         }
 
+        public void RequestStopAll()
```

Comment:
Per Joe:
- Only one instrumentation session can occur so if we have overlapping requests (old client, new client), we may get an error.

Reply:
Pending testing, basic parameter capture works as expected